README.md

```diff
@@ -24,6 +24,7 @@ command:
 $ go get github.com/tsenart/vegeta
```

Comment:
You can just change the initial paragraph to include the version constraint. **You need `go 1.5.2` installed (...)**